buildSrc/src/main/groovy/com/carrotsearch/gradle/junit4/RandomizedTestingPlugin.groovy

```diff
@@ -3,15 +3,14 @@ package com.carrotsearch.gradle.junit4
 import com.carrotsearch.ant.tasks.junit4.JUnit4
 import org.gradle.api.Plugin
 import org.gradle.api.Project
-import org.gradle.api.Task
 import org.gradle.api.tasks.TaskContainer
 
 class RandomizedTestingPlugin implements Plugin<Project> {
 
     void apply(Project project) {
-        setupSeed(project)
+        def seed = setupSeed(project)
```

Comment:
we try to use concrete types, as it makes it easier to port these classes to java (which we have slowly been doing).

Reply:
Then when we port from Java -> Kotlin we'll have to change back to `val` 😉

Reply:
☝️ joking about porting to Kotlin FYI 😆